app/views/spree/admin/variants/_edit_fields.html.erb

```diff
@@ -1,21 +1,14 @@
-<% content_for :page_actions do %>
-  <span id="new_add_volume_price" data-hook>
-    <%= button_link_to Spree.t(:add_volume_price), 'javascript:;', {
-      icon: 'add', :'data-target' => 'tbody#volume_prices', class: 'btn-success spree_add_fields', id: 'add_volume_price' } %>
-  </span>
-<% end %>
-
-<h3><%= Spree.t(:volume_prices) %></h3>
+<h3><%= I18n.t('spree.volume_prices') %></h3>
 <table class="table">
   <thead>
     <tr>
-      <th><%= Spree.t(:name) %></th>
```

Comment:
I think you should let translation keys being symbols, you can achieve that prepending `:`: `:'spree.name'`. Symbols are better for keys usage, also from a performance point of view:

```ruby
require 'benchmark'

Benchmark.bmbm do |x|
  x.report('string') { 100_000_000.times { 'spree.create' } }
  x.report('symbol') { 100_000_000.times { :'spree.create' } }
end
```

```
Rehearsal ------------------------------------------
string   5.953468   0.000366   5.953834 (  5.953751)
symbol   3.605581   0.000002   3.605583 (  3.605522)
--------------------------------- total: 9.559417sec

             user     system      total        real
string   5.605612   0.000000   5.605612 (  5.605510)
symbol   3.213020   0.000000   3.213020 (  3.212970)
```

Reply:
Very interesting @mdesantis :thinking:

Will fix as necessary, thanks!! :heart:

Reply:
> I think you should let translation keys being symbols, you can achieve that prepending :: :'spree.name'. 

We do that all over the code base, but great point @mdesantis although I am not a big fan of the proposed change as this looks a bit weird.

What about (the even more correct):

```rb
t(:name, scope: :spree)
```

_:point_up: this is actually what `Spree.t` used to do for us behind the scenes._

Reply:
@tvdeyen yeah that's even better